ci/jenkins/pipelines/prs/skuba-test.Jenkinsfile

```diff
@@ -52,32 +51,36 @@ pipeline {
         stage('Getting Ready For Cluster Deployment') { steps {
             sh(script: 'make -f skuba/ci/Makefile pre_deployment', label: 'Pre Deployment')
             sh(script: 'make -f skuba/ci/Makefile pr_checks', label: 'PR Checks')
-            sh(script: 'cd skuba; make install; cd ../', label: 'Install skuba')
+            sh(script: "pushd skuba; make -f Makefile install; popd", label: 'Build Skuba')
         } }
 
-        stage('Cluster Deployment') { steps {
-            sh(script: 'make -f skuba/ci/Makefile deploy', label: 'Deploy')
-            archiveArtifacts("skuba/ci/infra/${PLATFORM}/terraform.tfstate")
-            archiveArtifacts("skuba/ci/infra/${PLATFORM}/terraform.tfvars.json")
+        stage('Run Integration tests') { steps {
+            sh(script: "make -f skuba/ci/Makefile test_integration", label: "test_integration")
         } }
 
-        stage('Run end-to-end tests') { steps {
-           dir("skuba") {
-             sh(script: 'make build-ginkgo', label: 'build ginkgo binary')
-             sh(script: "make setup-ssh", label: "Setup SSH")
-             sh(script: "make test-e2e", label: "End-to-end tests")
-       } } }
     }
     post {
         always {
-            sh(script: 'make --keep-going -f skuba/ci/Makefile post_run', label: 'Post Run')
-            zip(archive: true, dir: 'testrunner_logs', zipFile: 'testrunner_logs.zip')
+            archiveArtifacts(artifacts: "skuba/ci/infra/${PLATFORM}/terraform.tfstate", allowEmptyArchive: true)
+            archiveArtifacts(artifacts: "skuba/ci/infra/${PLATFORM}/terraform.tfvars.json", allowEmptyArchive: true)
+            archiveArtifacts(artifacts: 'testrunner_logs/**/*', allowEmptyArchive: true)
+            junit('skuba/ci/infra/testrunner/*.xml')
+            sh(script: "make --keep-going -f skuba/ci/Makefile cleanup", label: 'Cleanup')
         }
         cleanup {
+            dir("${WORKSPACE}@tmp") {
+                deleteDir()
+            }
+            dir("${WORKSPACE}@script") {
+                deleteDir()
+            }
+            dir("${WORKSPACE}@script@tmp") {
```

Comment:
maybe I'm not understanding what's going on here but if we have already deleted `WORKSPACE/script` why delete `WORKSPACE/script/tmp`?

Reply:
AFAIR what @chentex told me, it was necessary because it had the `@` in its name, or something. it was found during manual runs on the jenkins worker itself that only like that the dirs get cleaned up